homeassistant/components/ultrasync/const.py

```diff
@@ -0,0 +1,20 @@
+"""Constants the Interlogix/Hills ComNav UltraSync Hub."""
+
+DOMAIN = "ultrasync"
+
+# Scan Time (in seconds)
+DEFAULT_SCAN_INTERVAL = 1
```

Comment:
Normally we don't allow lower scan interval than 5 seconds for polling integrations.